.changeset/unlucky-dots-judge.md

```diff
@@ -0,0 +1,60 @@
+---
+'@graphql-cli/codegen': minor
+'@graphql-codegen/cli': minor
+'@graphql-codegen/core': minor
+'@graphql-codegen/c-sharp': minor
+'@graphql-codegen/c-sharp-operations': minor
+'@graphql-codegen/flow': minor
+'@graphql-codegen/flow-operations': minor
+'@graphql-codegen/flow-resolvers': minor
+'@graphql-codegen/java-apollo-android': minor
+'@graphql-codegen/java-common': minor
+'@graphql-codegen/java': minor
+'@graphql-codegen/kotlin': minor
+'@graphql-codegen/java-resolvers': minor
+'@graphql-codegen/add': minor
+'@graphql-codegen/fragment-matcher': minor
+'@graphql-codegen/introspection': minor
+'@graphql-codegen/jsdoc': minor
+'@graphql-codegen/schema-ast': minor
+'@graphql-codegen/time': minor
+'@graphql-codegen/urql-introspection': minor
+'@graphql-codegen/visitor-plugin-common': minor
+'@graphql-codegen/typescript-apollo-angular': minor
+'@graphql-codegen/typescript-apollo-client-helpers': minor
+'@graphql-codegen/typescript-document-nodes': minor
+'@graphql-codegen/typescript-enum-array': minor
+'@graphql-codegen/typescript-generic-sdk': minor
+'@graphql-codegen/gql-tag-operations': minor
+'@graphql-codegen/typescript-graphql-files-modules': minor
+'@graphql-codegen/typescript-graphql-request': minor
+'@graphql-codegen/typescript-mongodb': minor
+'@graphql-codegen/named-operations-object': minor
+'@graphql-codegen/typescript-oclif': minor
+'@graphql-codegen/typescript-operations': minor
+'@graphql-codegen/typescript-react-apollo': minor
+'@graphql-codegen/typescript-react-offix': minor
+'@graphql-codegen/typescript-react-query': minor
+'@graphql-codegen/typescript-resolvers': minor
+'@graphql-codegen/typescript-rtk-query': minor
+'@graphql-codegen/typescript-stencil-apollo': minor
+'@graphql-codegen/typescript-type-graphql': minor
+'@graphql-codegen/typed-document-node': minor
+'@graphql-codegen/typescript': minor
+'@graphql-codegen/typescript-urql': minor
+'@graphql-codegen/typescript-urql-graphcache': minor
+'@graphql-codegen/urql-svelte-operations-store': minor
+'@graphql-codegen/typescript-vue-apollo': minor
+'@graphql-codegen/typescript-vue-apollo-smart-ops': minor
+'@graphql-codegen/typescript-vue-urql': minor
+'@graphql-codegen/gql-tag-operations-preset': minor
+'@graphql-codegen/graphql-modules-preset': minor
+'@graphql-codegen/import-types-preset': minor
+'@graphql-codegen/near-operation-file-preset': minor
+'@graphql-codegen/config-schema': minor
+'@graphql-codegen/testing': minor
```

Comment:
We don't publish testing and website to the NPM so those would changeset fail at NPM release. Could you remove these two?

Reply:
@ardatan done 👍 this issue in changesets has been open since so much time... https://github.com/atlassian/changesets/issues/436